### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@DeepDiver1975 requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

File: apps/user_ldap/command/setconfig.php [hunk 1]
```diff
@@ -73,8 +73,6 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 	/**
 	 * save the configuration value as provided
 	 * @param string $configID
-	 * @param string $configKey
-	 * @param string $configValue
```
Comment (LukasReschke) [inline apps/user_ldap/command/setconfig.php]: Needs fixing

File: apps/user_ldap/lib/jobs.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ static public function updateGroups() {
 	}
 
 	/**
-	 * @return int
+	 * @return string
```
Comment (LukasReschke) [inline apps/user_ldap/lib/jobs.php]: Not entirely true.

Comment (DeepDiver1975) [inline apps/user_ldap/lib/jobs.php]: use intval() within to enforce the proper type?

**Comment by DeepDiver1975** [comment]
@LukasReschke please have another look - THX

File: lib/private/backgroundjob/joblist.php [hunk 3]
```diff
@@ -68,7 +68,7 @@ public function add($job, $argument = null) {
 	}
 
 	/**
-	 * @param Job|string $job
```
Comment (LukasReschke) [inline lib/private/backgroundjob/joblist.php]: Can also be string for whatever reason :speak

File: apps/user_ldap/lib/jobs.php (hunk 2 above)
Comment (MorrisJobke) [inline apps/user_ldap/lib/jobs.php]: Can the type then be adjusted?

File: lib/private/backgroundjob/joblist.php (hunk 3 above)
Comment (MorrisJobke) [inline lib/private/backgroundjob/joblist.php]: @DeepDiver1975 Please fix this too.

**Comment by DeepDiver1975** [comment]
I'll resubmit this ....

**Pull request closed without merging**